### ISSUE TITLE
Upgrading to PhantomJS 2.0

### ISSUE THREAD

**Issue by symi** [mmanela/chutzpah#324]
Enhancement/Feature Request.
Since PhantomJS 2.0 is out and stable, upgrading chutzpah to use the latest version will solve many poly-filling problems.

**Comment by mmanela** [comment]
Thanks for reporting. Will be sure to include this in next release

**Comment by mthamil** [comment]
Just thought I would add that I finally discovered that an issue I was having with tests failing within Visual Studio but not in the browser had to do with the fact that phantomjs didn't support Map. Does 2.0 add support for it?

**Comment by gurumusch** [comment]
I think it would be great if we could also be able to run tests with the old phantomjs version. We have several thousand tests running with chutzpah and we will need to check if they work well with phantomjs 2.

**Comment by benkaraban** [comment]
I had to upgrade to PhantomJS 2.0.0 because 1.x versions were crashing randomly when running our tests. The 2.0 version introduced some breaking changes and I had to tweak the chutzpahRunner.js file to make it work with the new version. I can provide you the updated file to help you with the PhantomJS upgrade.

Note I'm not using a full/real version of chutzpah, we've just hacked it in our build system and that's what I had to fix. I guess you'll probably have to change more things to make it work end to end.
